src/main/java/com/chargerrobotics/commands/shooter/HoodAutoCommand.java

```diff
@@ -9,21 +9,40 @@
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
 import com.chargerrobotics.subsystems.LimelightSubsystem;
+import com.chargerrobotics.subsystems.ShooterHoodSubsystem;
 
 public class HoodAutoCommand extends CommandBase {
+  private final ShooterHoodSubsystem shooterSubsystem;
+  private boolean isOpening;
   /**
    * Creates a new HoodAutoCommand.
    * 
    * Using the distance the robot is from the target, set the hood angle.
    * f(distance)
    */
-  public HoodAutoCommand() {
+  public HoodAutoCommand(ShooterHoodSubsystem shooterSubsystem) {
+    this.shooterSubsystem = shooterSubsystem;
     // Use addRequirements() here to declare subsystem dependencies.
   }
 
   // Called when the command is initially scheduled.
   @Override
   public void initialize() {
+    System.out.println("hood auto command init");
+    // Angle to target
+    double angle = shooterSubsystem.getAngleByZone();
+    // Ticks to Angle
+    double newposition = shooterSubsystem.findHoodTargetTicks(angle);
+    // Current Position
+    double position = shooterSubsystem.getHoodPosition();
+    // Getting Direction
+    if (position < newposition) {
+       isOpening = true;
+    } else {
+      isOpening = false;
+    }
+    // Setting Motor Speed
+    shooterSubsystem.setHoodSpeed(isOpening ? -0.25 : 0.25);
```

Comment:
How do we know that the hood has been calibrated? If it hasn't, then this operation could take you to the incorrect position.